src/Dropdown/helpers.ts

```diff
@@ -1,5 +1,10 @@
 import { Positioning } from '../useSnapEdgeToParent'
 
-export const makeCssPosition = ({ horizontal, vertical }: Positioning, height: number) =>
+export interface Dimensions {
+  height: number
+  width: number
+}
+
+export const makeDefaultCSS = ({ horizontal, vertical }: Positioning, _: Dimensions, { height }: Dimensions) =>
```

Comment:
Since the second and third argument are both Dimensions I think it would be better to change the parameters to:
```typescript
const makeDefaultCSS = (
    { horizontal, vertical }: Positioning, 
    targetDimensions: Dimensions, 
    parentDimensions: Dimensions
) => {}
```